### ISSUE TITLE
How do I set the pivot point to be the handle of the slider. The slider still works if someone drags the center of the slider.

### ISSUE THREAD

**Issue by mayankdklord** [ThunderStruct/MSCircularSlider#18]

**Comment by MoonshineSG** [comment]
similar issue here.... can the slider only be "active" when touching the actual handle not teh rest of the slider surface ? even a short delay woudl be good: touch, hold a seconds (haptic feedback?) and then start sliding... this would prevent accidental changes in value (in my case a "pull to refresh" sometimes triggers slider value change... )

**Comment by MoonshineSG** [comment]
found my solution: changes made in MSCircularSlider about line 485:

```
override public func beginTracking(_ touch: UITouch, with event: UIEvent?) -> Bool {
        let location = touch.location(in: self)
        
        if handle.contains(location) {
            handle.isPressed = true
            delegate?.circularSlider(self, startedTrackingWith: currentValue)
            if #available(iOS 10.0, *) {
                UIImpactFeedbackGenerator(style: .medium).impactOccurred()
            }
            setNeedsDisplay()            
            return true
        }
        
        return false //pointInsideCircle(location)
    }
```

**Comment by endy-s** [comment]
@MoonshineSG Thanks for the solution! Just opened the PR :)

**Comment by ThunderStruct** [comment]
Thanks @MoonshineSG and @endy-s for the solution and the PR!
Fixed in #24
